docs/src/pages/components/slider-styled/DiscreteSlider.tsx

```diff
@@ -1,23 +1,19 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import Slider from '@material-ui/lab/SliderStyled';
 
-const useStyles = makeStyles({
-  root: {
-    width: 300,
-  },
-});
+const Root = styled('div')`
+  width: 300px;
+`;
 
 function valuetext(value: number) {
   return `${value}°C`;
 }
 
 export default function DiscreteSlider() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Root>
```

Comment:
Maybe we could use a box here

```suggestion
    <Box width={300}>
```

Reply:
Good idea, let me refactor